src/wallet/wallet.cpp

```diff
@@ -3114,6 +3114,9 @@ bool CWallet::SelectCoinsGroupedByAddresses(std::vector<CompactTallyItem>& vecTa
             isminefilter mine = ::IsMine(*this, txdest);
             if(!(mine & filter)) continue;
 
+            CompactTallyItem& item = mapTally[txdest];
```

Comment:
This might end up with empty/uninitialized items in `mapTally` when the following `if then continue` statements skip the item. Better to test for existence of the items before accessing it here.

Reply:
🙈 

Applied suggested fixes (with few slight modifications).